crates/core/src/client/tests/mod.rs

```diff
@@ -194,105 +109,3 @@ async fn test_gas_price() {
     // Then
     assert!(gas_price > U256::from(0));
 }
-
-#[tokio::test]
-async fn test_get_logs_from_bigger_than_current() {
-    // Given
-    let fixtures = fixtures(vec![wrap_kakarot!(JsonRpcMethod::BlockNumber)]);
-    let client = init_mock_client(Some(fixtures));
-    let filter = Filter {
-        block_option: FilterBlockOption::Range {
-            from_block: Some(BlockNumberOrTag::Number(19641)),
-            to_block: Some(BlockNumberOrTag::Number(19642)),
-        },
-        ..Default::default()
-    };
-
-    // When
-    let logs = client.get_logs(filter).await.unwrap();
-
-    // Then
-    assert_eq!(logs, FilterChanges::Empty);
-}
-
-#[tokio::test]
-async fn test_get_logs_to_less_than_from() {
-    // Given
-    let fixtures = fixtures(vec![wrap_kakarot!(JsonRpcMethod::BlockNumber)]);
-    let client = init_mock_client(Some(fixtures));
-    let filter = Filter {
-        block_option: FilterBlockOption::Range {
-            from_block: Some(BlockNumberOrTag::Number(2)),
-            to_block: Some(BlockNumberOrTag::Number(1)),
-        },
-        ..Default::default()
-    };
-
-    // When
-    let logs = client.get_logs(filter).await.unwrap();
-
-    // Then
-    assert_eq!(logs, FilterChanges::Empty);
-}
-
-#[tokio::test]
-async fn test_get_logs() {
```

Comment:
A bit unlucky that we loose this test, but I'm thinking of:
- For integration: use hive
- For e2e, we can use Katana and go through all the steps in a tx -> run a deploy tx for a counter sc, check logs, call the count() method, increment the counter, call the count() method, assert

Reply:
Replaced in eth-rpc -> e2e test for get_logs